cymetric/metrics.py

```diff
@@ -322,6 +326,47 @@ def fco_u_mined(series):
 del _udeps, _uschema
 
 
+# SWU Required [million SWU]
+_swudeps = [
+    ('Materials', ('ResourceId', 'ObjId', 'TimeCreated', 'NucId'), 'Mass'),
+    ('Transactions', ('ResourceId',), 'Commodity')
+    ]
+
+_swuschema = [('Year', ts.INT), ('SWU', ts.DOUBLE)]
+
+@metric(name='FcoSwu', depends=_swudeps, schema=_swuschema)
+def fco_swu(series):
+    """FcoSwu metric returns the separative work units required for each 
+    year in a 200-yr simulation. This is written for FCO databases that 
+    use the Bright-lite (i.e., the U235 and U238 are given separately 
+    in the FCO simulations).
+    """
+    tools.raise_no_pyne('SWU Required could not be computed', HAVE_PYNE)
+    mass = pd.merge(series[0].reset_index(), series[1].reset_index(),
+            on=['ResourceId'], how='inner').set_index(['ObjId', 'TimeCreated', 'NucId'])
+    swu = []
+    prods = {}
+    mass235 = {}
+    m = mass[mass['Commodity'] == 'LWR Fuel']
+    for (obj, _, nuc), value in m.iterrows():
+        prods[obj] = prods.get(obj, 0.0) + value['Mass']
+        if nuc == 922350000:
+            mass235[obj] = value['Mass']
+    for obj, m235 in mass235.items():
+        x_prod = m235 / prods[obj]
+        swu0 = enr.swu(0.0072, x_prod, 0.0025, product=prods[obj]) / 1e6
```

Comment:
To some degree this is also a question about the generality of this method.  While you wrote it for the FCO work specifically, it might have broader applicability.  There are at least 2 places for the tails enrichment to come from:
1. the db itself
2. user input

It may be out of scope to do that right now, but I thought I'd raise the question.